experimenter/experimenter/experiments/api/v5/types.py

```diff
@@ -662,3 +669,9 @@ def resolve_languages(self, info):
 
     def resolve_changes(self, info):
         return self.changes.all().order_by("changed_on")
+
+    def resolve_excluded_experiments(self, info):
+        return self.excluded_experiments.only("id", "slug")
+
+    def resolve_required_experiments(self, info):
+        return self.required_experiments.only("id", "slug")
```

Comment:
Slug and not name?  Name might be more humane?

Reply:
Also above it says it's a list of ExperimentType but here we're only providing id and slug, would it make sense to add a type that's just those two fields?

Reply:
The .only() is just a DB optimization because we know in practice the query is just ehse fields. If the query asks for more fields, the model will query for them on-the-fly.

Reply:
Also turns out the query was asking for name and publicDescription already, so I've added those to the query and will update this soon.